app/lib/actions/pulp3/content_view_version/export.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Pulp3
+    module ContentViewVersion
+      class Export < Pulp3::AbstractAsyncTask
+        input_format do
+          param :smart_proxy_id, Integer
+          param :content_view_version_id, Integer
+          param :export_base_dir, String
+        end
+
+        def export(cvv)
+          smart_proxy = SmartProxy.find(input[:smart_proxy_id])
+          api = ::Katello::Pulp3::Api::Core.new(smart_proxy)
+          export_base_dir = input[:export_base_dir]
```

Comment:
curious, does this need to be overridden in the action?  It looks like it comes from a setting, could that just be looked up?

Reply:
updated. Moved it to the service class